### DISCUSSION TITLE
paddlelabel kie标注

### DISCUSSION THREAD

**Discussion by Gpy23** [PaddlePaddle/PaddleOCR#12777] [Q&A]
kie标注的时候，如下图所示，一个question对应很多行answer，是每一行（或者一行里还要标成两个标注块，因为字数过多）都标成answer并赋予一个id，且link的时候，question和每一行answer都link上吗？
<img width="606" alt="微信图片_20240607113842" src="https://github.com/PaddlePaddle/PaddleOCR/assets/73943977/bfbbabf1-0f47-4195-bfdc-88174290be76">

**Comment by GreatV** [comment]
please refer:
- https://github.com/PFCCLab/PPOCRLabel/discussions/14
